pyproject.toml

```diff
@@ -20,6 +20,7 @@ python = ">=3.7.0,<4.0"
 python-dateutil = "^2.8.2"
 requests = "^2.28.2"
 urllib3 = "^1.26.9"
+fire = "^0.5.0"
```

Comment:
I would recommend using [typer](https://typer.tiangolo.com/tutorial/) instead -- I'm not sure `fire` is actively maintained.